frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -130,29 +128,25 @@ export default memo(function HeatMap(props: Props): JSX.Element {
             (isAnyTissueLoading(isLoading) && (
               <ChartWrapper
                 top={xAxisHeight}
-                hidden={!isAnyTissueLoading(isLoading)}
+                visible={isLoadingAPI || isAnyTissueLoading(isLoading)}
```

Comment:
I'm still looking at this code, but thought I'd raise this first - it seems like when `isLoadingAPI` is `true`, the loading skeleton component doesn't get rendered, so do we need `isLoadingAPI` in `visible`?

<img width="808" alt="Screenshot 2023-12-19 at 8 37 10 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/09c7f6a6-6a36-47e5-ab78-b473f8c2180f">

Reply:
removed